codespell_lib/data/dictionary.txt

```diff
@@ -10261,6 +10359,8 @@ cheecksum->checksum
 cheecksums->checksums
 cheeper->cheaper
 cheeta->cheetah
+cheetsheet->cheatsheet
+cheetsheets->cheatsheets
```

Comment:
Perhaps add `cheet`/`cheets`/`cheeted`/`cheeting` as typos of `cheat`, `sheet`, `cheek`, `cheep`, cheer, etc.